helm-charts/seldon-core-analytics/Chart.lock

```diff
@@ -0,0 +1,9 @@
+dependencies:
+- name: grafana
+  repository: https://kubernetes-charts.storage.googleapis.com
+  version: 4.2.3
```

Comment:
The current grafana is major version 6 I think. Are we changing it? 
https://github.com/SeldonIO/seldon-core/issues/540

Reply:
I think I was looking at the chart version instead of the application version of Grafana: https://hub.helm.sh/charts/stable/grafana which is currently 4... but I will check that it definitely is the chart version and not the Grafana application version that is required.

Reply:
If I change version to 6 I get following error:

> Error: can't get a valid version for repositories grafana. Try changing the version constraint in Chart.yaml

So I believe it is referring to chart version and not application version.

Reply:
Okay, great can you also confirm the application version, just to be sure.

Reply:
6.5.2